src/components/consul/detail/ConsulPublicAuthorComponent.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Wrapper, WrapperRow } from '../../Wrapper';
+import { texts } from '../../../config';
+import { RegularText } from '../../Text';
+import { momentFormatUtcToLocal } from '../../../helpers';
+
+const text = texts.consul;
+
+export const ConsulPublicAuthorComponent = ({ authorData }) => {
+  const { publicAuthor, commentsCount, publicCreatedAt } = authorData;
+
+  return (
+    <Wrapper>
+      <WrapperRow>
+        <RegularText primary>{publicAuthor ? publicAuthor.username : 'Privat'}</RegularText>
+        <RegularText> · </RegularText>
+        <RegularText smallest>{momentFormatUtcToLocal(publicCreatedAt)}</RegularText>
```

Comment:
is there always a `publicCreatedAt`?

Reply:
added and updated the necessary control to the code.